docker/README.md

```diff
@@ -0,0 +1,17 @@
+## Build
+To use `cross` to compile for Raspberry Pi you first need to build the docker image.
+Use these commands to build the images needed.
+
+**NOTE:** Run these commands inside the `docker` folder. This is needed since `docker`
+uses surrounding directories as "context" for building the image, which means it'll
+copy the entire `target` directory.
+
+### Raspberry Pi 2/3/4 (32 bits)
+```
+$ docker build -t iced-rs/armv7 -f Dockerfile.armv7-unknown-linux-gnueabihf .
+```
+
+### Raspberry Pi 3/4 (64 bits)
+```
+$ docker build -t iced-rs/aarch64 -f Dockerfile.aarch64-unknown-linux-gnu .
+```
```

Comment:
I think this `docker` folder is generic enough to become a standalone repository for all of the Rust ecosystem!

Instead of including it here, I believe we could create a GitHub Action to easily cross-compile any Rust project for Raspberry Pis. I'll gladly help you out with this!

Once we do that, we could then change the `build` workflow to additionally build binaries of the `todos` example for the Raspberry Pis and make sure we keep everything compatible moving forward.

Reply:
Right now the `Dockerfile`s are tailored to `iced`, but if we wanted we could upload the images to 'Docker Hub'.
This could be useful to developers cross-compiling their `iced` apps, meaning they could build using our `Dockerfile`:
```Dockerfile
FROM iced-rs/iced:arm64
```

Reply:
I like the idea! I have created [an organization in Docker Hub](https://hub.docker.com/orgs/icedrs/repositories) and [a new `iced-rs/docker` repository](https://github.com/iced-rs/docker).

Let me know your username in Docker Hub and I will add you.

Reply:
I have added [a workflow in `iced-rs/docker` to build the images in GitHub CI and push them to Docker Hub](https://github.com/iced-rs/docker/blob/4a5994498743955fd730048ec0fe3d672b061afa/.github/workflows/publish.yml).

Also, I have removed the `docker` folder in 2b3528e9929e1b4bb563348cd5f694b83d0caec4 and added a `todos_raspberry` job to the `build` workflow in 522368e8af6947ea0676f62bf1ae5f68acb2058d. [Everything seems to work nicely!](https://github.com/iced-rs/iced/runs/4880885146?check_suite_focus=true)

It'd be great if you could try the binaries from this GitHub CI run in the Pis: https://github.com/iced-rs/iced/actions/runs/1722948462